Maestro/subscriptions.json

```diff
@@ -984,7 +984,8 @@
           "GithubRepoOwner": "aspnet",
           "GithubRepoName": "<trigger-repo>",
           "HeadBranch": "<trigger-branch>",
-          "BaseBranch": "release/3.0"
+          "BaseBranch": "release/3.0",
+          "ExtraSwitches": "-QuietComments"
```

Comment:
This is in @vijayrkn's world, not ours'. Sorry if I pointed you here @JamesNK

Reply:
Removed